Cargo.toml

```diff
@@ -147,7 +147,8 @@ default-features = false
 [dev-dependencies]
 # See https://bheisler.github.io/criterion.rs/book/user_guide/known_limitations.html for the usage
 # of `real_blackbox` feature.
-criterion = { version = "0.2.11", features=['real_blackbox'] }
+criterion = "0.3"
+criterion-cpu-time = { git = "https://github.com/YangKeao/criterion-cpu-time.git", branch = "master" }
```

Comment:
maybe you can consider publish this crate

Reply:
Yes! But after it gets more stable.

Reply:
seem you can use a stack var instead of mallocing every time in get user CPU time. 

see https://github.com/anp/rust-runtime-benchmarks/blob/master/bench-suite-linux/src/bencher.rs#L3

Reply:
@breeswish It has been published on [crates.io](https://crates.io/crates/criterion-cpu-time) :smile: